docker-compose/docker-compose.yml

```diff
@@ -85,15 +84,50 @@ services:
 
   seq:
     image: datalust/seq:latest
+    container_name: seq
     ports:
-      - "8081:80"
+      - "8082:80"
       - "5341:5341"
     volumes:
       - seq-data:/data
     environment:
       - ACCEPT_EULA=Y
     restart: unless-stopped
 
+  adminApi:
+    image: nmshd-adminapi
+    container_name: adminApi
+    hostname: adminApi
```

Comment:
Can you convert this to lowercase please? (same for the consumer api; don't know why it is camel case)